chroma_core/services/lustre_audit/update_scan.py

```diff
@@ -218,17 +218,19 @@ def update_target_mounts(self):
                         ["mounted", "unmounted"],
                     )
 
-            if target_mount.target.active_mount == None:
-                TargetRecoveryInfo.update(target_mount.target, {})
-                TargetRecoveryAlert.notify(target_mount.target, False)
+            if target_mount.target.active_mount is None:
```

Comment:
Should this check be inside the atomic transaction?

Reply:
Like this?

```python
with transaction.atomic():
    if target_mount.target.active_mount is None:
        TargetRecoveryInfo.update(target_mount.target, {})
        TargetRecoveryAlert.notify(target_mount.target, False)
    elif mounted_locally:
        recovering = TargetRecoveryInfo.update(target_mount.target, recovery_status)
        TargetRecoveryAlert.notify(target_mount.target, recovering)
```

Reply:
I think so.

Reply:
Sure, I'll update.